environments/stg/backend_lb_config.yaml

```diff
@@ -253,6 +253,7 @@ gateways:
         ssl_enabled: true
       - product: idam
         component: user-dashboard
+        cookie_based_affinity: Enabled
```

Comment:
Why?

Reply:
Removed this now, https://github.com/hmcts/azure-platform-terraform/pull/972/commits/dbd3983e275c1483f5963b56f7931dffe0f76f44 - was just leftover from originally duplicating all of idam-web-admin settings :)